src/feature/gui/gui-runner.c

```diff
@@ -51,6 +51,7 @@ enum {
 enum {
 	SCREENSHOT_VALID = 0x10000,
 	SCREENSHOT_INVALID = 0x20000,
+	SCREENSHOT_STALE = 0x30000,
```

Comment:
You don't need a new value for this, you can just use INVALID.

Reply:
Yeah, I also saw that I could use `INVALID`, especially cause `SCREENSHOT_VALID | SCREENSHOT_INVALID` is the same as `SCREENSHOT_STALE`. I figured I'd be 'cleaner' and use a dedicated enum...

I can make the change to just use `INVALID` though, no problem.